2021-11-21 22:09:37,952 INFO    MainThread:79288 [wandb_setup.py:_flush():71] setting env: {'project': '1018_lw'}
2021-11-21 22:09:37,952 INFO    MainThread:79288 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-21 22:09:37,952 INFO    MainThread:79288 [wandb_init.py:_log_setup():371] Logging user logs to /home/yu-jw19/venom/project2/wandb/run-20211121_220937-4kz9b5k4/logs/debug.log
2021-11-21 22:09:37,952 INFO    MainThread:79288 [wandb_init.py:_log_setup():372] Logging internal logs to /home/yu-jw19/venom/project2/wandb/run-20211121_220937-4kz9b5k4/logs/debug-internal.log
2021-11-21 22:09:37,953 INFO    MainThread:79288 [wandb_init.py:init():404] calling init triggers
2021-11-21 22:09:37,953 INFO    MainThread:79288 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-21 22:09:37,954 INFO    MainThread:79288 [wandb_init.py:init():449] starting backend
2021-11-21 22:09:37,954 INFO    MainThread:79288 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-21 22:09:37,981 INFO    MainThread:79288 [backend.py:ensure_launched():199] starting backend process...
2021-11-21 22:09:37,998 INFO    MainThread:79288 [backend.py:ensure_launched():204] started backend process with pid: 79489
2021-11-21 22:09:38,000 INFO    MainThread:79288 [wandb_init.py:init():458] backend started and connected
2021-11-21 22:09:38,001 INFO    MainThread:79288 [wandb_init.py:init():519] updated telemetry
2021-11-21 22:09:38,051 INFO    MainThread:79288 [wandb_init.py:init():549] communicating current version
2021-11-21 22:09:39,274 INFO    MainThread:79288 [wandb_init.py:init():554] got version response 
2021-11-21 22:09:39,274 INFO    MainThread:79288 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-21 22:09:39,815 INFO    MainThread:79288 [wandb_init.py:init():592] starting run threads in backend
2021-11-21 22:09:40,639 INFO    MainThread:79288 [wandb_run.py:_console_start():1816] atexit reg
2021-11-21 22:09:40,640 INFO    MainThread:79288 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-21 22:09:40,640 INFO    MainThread:79288 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-21 22:09:40,643 INFO    MainThread:79288 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-21 22:09:40,643 INFO    MainThread:79288 [wandb_init.py:init():619] run started, returning control to user process
2021-11-21 22:09:40,652 INFO    MainThread:79288 [wandb_run.py:_config_callback():962] config_cb None None {'dataset': 'emotiondataset', 'name': 'first_test_network==32', 'no': '0', 'model': 'resnet', 'print_freq': 10, 'layers': 32, 'droprate': 0.3, 'cardinality': 8, 'growth_rate': 12, 'compression_rate': 0.5, 'bn_size': 4, 'alpha': 200, 'N': 2, 'M': 10, 'augment': True, 'randaugment': True, 'checkpoint': 'checkpoint', 'resume': False, 'cos_lr': True, 'initial_learning_rate': 0.1, 'changing_lr': [80, 120], 'en_wandb': True, 'warm_up': False, 'exp_id': '00012_network==32', 'commit': 'not_set'}
2021-11-21 22:10:09,197 INFO    MainThread:79288 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-21 22:10:09,198 INFO    MainThread:79288 [wandb_run.py:_restore():1758] restore
2021-11-21 22:10:11,661 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-21 22:10:12,246 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-21 22:10:12,859 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:12,963 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:13,066 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:13,170 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:13,273 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:13,377 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:13,481 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6743
  total_bytes: 8999
}

2021-11-21 22:10:13,584 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:13,688 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:13,792 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:13,895 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:13,999 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:15,292 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:16,347 INFO    MainThread:79288 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}
local_info {
}

2021-11-21 22:10:17,454 INFO    MainThread:79288 [wandb_run.py:_append_files():2184] logging synced files
